### DISCUSSION TITLE
在web网页中使用自带得备份与升级功能进行升级会丢失root目录下得内容请问怎么解决

### DISCUSSION THREAD

**Discussion by rianjskis** [immortalwrt/immortalwrt#1597] [Q&A]
以前这样在线更新并不会丢失root目录下的文件,因为我使用了acme申请证书,默认在root目录下,每次更新的话都会丢失需要重新安装acme申请证书,请问是哪里做了改动吗,使用的是23.05版本R2S路由器.

**Comment by metaforget** [comment]
你有没有在配置里加/root目录啊，我现在加了也会丢
